### PR TITLE
converted files to haml and added home controller and view

### PR DESCRIPTION
**Pull request by AbdullahAlger**

### CODE REVIEW

**Comment by AbdullahAlger** [comment]
Had to take out webruby gem because it wanted me to install emscripten inside the rails app. Perhaps we could discuss this, or I could just hijack the webruby.js file from the git repo I sent an email about recently.

**Comment by eliotsykes** [comment]
This might help out, looks like an up-to-date version of mruby.js, https://github.com/xxuejie/mruby.vm.js

File: app/assets/javascripts/home.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (eliotsykes) [inline app/assets/javascripts/home.js]: You might want to add config to `config/application.rb` to stop these blank assets from being generated automatically and adding clutter.

File: app/helpers/home_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module HomeHelper
```
Comment (eliotsykes) [inline app/helpers/home_helper.rb]: Can be safely deleted.

File: spec/helpers/home_helper_spec.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```
Comment (eliotsykes) [inline spec/helpers/home_helper_spec.rb]: Can be safely deleted.

**Pull request merged**